app/routes/questionnaire.py

```diff
@@ -331,6 +335,10 @@ def send_confirmation_email(session_store, schema):
                 email=confirmation_email.get_url_safe_serialized_email(),
             )
         )
+    else:
+        logger.info(
+            "email validation error", error_message=confirmation_email.form.errors
+        )
```

Comment:
This would be logged for all non POST requests. It should be:
```py
if method == "POST":
	if form_valid:
		return redirect()
	
	log_error()

handle_get()